solidity/contracts/tutorials/Storage.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title Storage
+ * @dev Store & retrieve value in a variable
+ */
+
+contract Storage {
+
+    uint256 number;
```

Comment:
Might want to be explicit that this is internal with `uint256 internal _number` (leading underscore is the [preferred style](https://docs.soliditylang.org/en/latest/style-guide.html#underscore-prefix-for-non-external-functions-and-variables) for internal variables).